puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/katello/3.8.groovy

```diff
@@ -0,0 +1,42 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Setup Environment') {
```

Comment:
Now that we're copying this between pipelines, can we introduce a helper for this?

Reply:
@zjhuntin and I discussed this and he is going to look at re-factoring but outside the scope of this change since that would hol dup this needed pipeline.

Reply:
Would https://github.com/theforeman/foreman-infra/pull/736 be a better place?

Reply:
Yup -- thats where he plans to test re-factoring out.